<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CParameterWakeOnLanUI</class>
 <widget class="QWidget" name="CParameterWakeOnLanUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="gbWakeOnLan">
     <property name="title">
      <string>Wake on lan</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Mac:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leMac">
          <property name="inputMask">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Board address:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leBroadcastAddress">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbPort">
          <property name="minimum">
           <number>7</number>
          </property>
          <property name="maximum">
           <number>9</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>9</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lePassword">
          <property name="echoMode">
           <enum>QLineEdit::EchoMode::Password</enum>
          </property>
          <property name="placeholderText">
           <string>A password string of length 6. or Hexadecimal representation of 6 bytes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbShow">
          <property name="toolTip">
           <string>Menu: Tools→Settings→Client→Password-&gt;Enable view password</string>
          </property>
          <property name="statusTip">
           <string>Menu: Tools→Settings→Client→Password-&gt;Enable view password</string>
          </property>
          <property name="whatsThis">
           <string>Menu: Tools→Settings→Client→Password-&gt;Enable view password</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="eye-on"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbSave">
          <property name="toolTip">
           <string>Menu: Tools→Settings→Client→Password-&gt;Enable Save password</string>
          </property>
          <property name="statusTip">
           <string>Menu: Tools→Settings→Client→Password-&gt;Enable Save password</string>
          </property>
          <property name="whatsThis">
           <string>Menu: Tools→Settings→Client→Password-&gt;Enable Save password</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="document-save"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Repeat:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbRepeat"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Interval:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbInterval">
          <property name="maximum">
           <number>3000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>ms</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Delay:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbDelay">
          <property name="maximum">
           <number>600</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>s</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
